Content.Shared/Storage/Components/EntityStorageComponent.cs

```diff
@@ -8,8 +8,8 @@
 
 namespace Content.Shared.Storage.Components;
 
-[NetworkedComponent]
-public abstract partial class SharedEntityStorageComponent : Component
+[RegisterComponent, NetworkedComponent]
+public partial class EntityStorageComponent : Component
```

Comment:
If you're renaming it then move the client / server stuff and seal it please.

Reply:
There is no client / server stuff left to move here so I've just sealed it.